### PR TITLE
Refactor --sort logic, nicer error messages

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: todoman/main.py [hunk 1]
```diff
@@ -39,6 +43,10 @@ def get_task_sort_function(fields):
             '-created_at',
         ]
 
+    for field in fields:
+        if not hasattr(model.Todo, field):
```
Comment (hobarrera) [inline todoman/main.py]: This would seem to fail for "negative fields", eg: `-priority`.

Comment (untitaker) [inline todoman/main.py]: Yes, just noticed.

**Comment by hobarrera** [comment]
LGTM. Feel free to merge when tests complete.

**Comment by untitaker** [comment]
Ah, this is buggy as well if the property is None.

**Comment by untitaker** [comment]
Turns out the logic isn't that fixable, we still have the hardcoded specialcases.

**Pull request merged**